condor/README-xml.md

```diff
@@ -105,3 +105,60 @@ Here are sample XML records sent for ResourceType = Batch and BatchPilot
 </JobUsageRecord>
 ```
 
+#### Notes
+
+The key difference is the "payload" jobs (ResourceType=Payload in GRACC, ResourceType=BatchPilot in the xml sent by the
+gratia-probe) are identified when they have a MATCH_EXP_JOBGLIDEIN_ResourceName attribute in their classad, as found in
+the condor history logfile.
```

Comment:
We may as well reference where that configuration comes from: https://github.com/opensciencegrid/gratia-probe/blob/master/condor/99_gratia-gwms.conf

Reply:
The condor probe normally reads from the per job history directory.  Not directly from `condor_history`, though is has the capability to read from `condor_history` (history logfile).

Reply:
Should we just say "as found in the condor history" ?

Reply:
Maybe, "As found in the HTCondor per job history file"